plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectWizard.java

```diff
@@ -84,7 +99,7 @@ private void retrieveConfigurationValues() {
       config.setEclipseProjectLocationUri(appEnginePage.getLocationURI());
     }
 
-    config.setAppEngineLibraries(appEnginePage.getSelectedLibraries());
+    config.setAppEngineLibraries(librariesPage.getSelectedLibraries());
```

Comment:
We should rename this to `setSelectedLibraries` (and all related changes in `AppEngineProjectConfig`). Doesn't have to be in this PR.